src/Grid/grid.jl

```diff
@@ -100,27 +123,53 @@ end
 ##########################
 # Grid utility functions #
 ##########################
-@inline getdim(::AbstractGrid{dim}) where {dim} = dim
+@inline getdim(grid::AbstractGrid{dim}) where {dim} = dim
+"""
+    getcells(grid::AbstractGrid) 
+    getcells(grid::AbstractGrid, v::Union{Int,Vector{Int}} 
+    getcells(grid::AbstractGrid, set::String)
+
+Returns either all `cells::Vector{C<:AbstractCell}` of a `grid` or a subset based on an `Int`, `Vector{Int}` or `String`.
+Whereas the last option tries to call a `cellset` of the `grid`.
+"""
 @inline getcells(grid::AbstractGrid) = grid.cells
 @inline getcells(grid::AbstractGrid, v::Union{Int, Vector{Int}}) = grid.cells[v]
 @inline getcells(grid::AbstractGrid, set::String) = grid.cells[collect(grid.cellsets[set])]
+"Returns and `Int` corresponding to how many cells are in the `grid`."
 @inline getncells(grid::AbstractGrid) = length(grid.cells)
+"Returns the celltype of the `grid`."
 @inline getcelltype(grid::AbstractGrid) = eltype(grid.cells)
 @inline getcelltype(grid::AbstractGrid, i::Int) = typeof(grid.cells[i])
 
+"""
+    getnodes(grid::AbstractGrid) 
+    getnodes(grid::AbstractGrid, v::Union{Int,Vector{Int}}
+    getnodes(grid::AbstractGrid, set::String)
+
+Returns either all `nodes::Vector{Node{dim,T}}` of a `grid` or a subset based on an `Int`, `Vector{Int}` or `String`.
+The last option tries to call a `nodeset` of the `grid`.
```

Comment:
Do we want to be this strict in the interface? Maybe it should be more relaxed and ungrouped? E.g. for ` getnodes(grid::AbstractGrid, v::Union{Int,Vector{Int}} ` something like "Returns a node with given index or a range of nodes for a given index range."